src/docs/config/overview.md

```diff
@@ -119,17 +125,27 @@ During production builds, the content of each generated file is hashed to repres
 hashFileNames: true
 ```
 
+## invisiblePrehydration
 
-## hashedFileNameLength
-
-*default: `8`*
+*default: `true`*
 
-When the `hashFileNames` config is set to `true`, and it is a production build, the `hashedFileNameLength` config is used to determine how many characters the file name's hash should be.
+When true, `invisiblePrehydration` will use Ionic's opinion to visually hide components before they are hydrated by adding an automatically injected style tag to the document's head. Setting `invisiblePrehydration` to `false` will not inject the style tag into the head, allowing you to style your web components pre-hydration. Please note that setting `invisiblePrehydration` to false will cause everything to be visible when your page is loaded, causing more prominent Flash of Unstyled Content (FOUC), however you can style your web component's fallback content to your preference.
```

Comment:
idk if we want to just leave this alone since it was just moved, but "Ionic's opinion" here strikes me as a really strange phrase - I would have no idea what that means if I came across it I think

Reply:
Agree this is super goofy. I'll go ahead and rephrase this slightly to get rid of that phrase while I'm in here